CondFormats/GeometryObjects/interface/PHGCalParameters.h

```diff
@@ -65,7 +65,7 @@ class PHGCalParameters {
   std::vector<int>         layerGroupO_;
   std::vector<uint32_t>    trformIndex_;
   double                   waferR_;
-  double                   slopeMin_;
+  std::vector<double>      slopeMin_;
```

Comment:
@bsunanda @ianna this CondFormats object encoding the geometry has not been used for DB payloads so far, is this correct? So this change does not break any backward compatibility

Reply:
Yes it is correct. We have not put the Phase2 geometries in DB yet.